docs/sources/enterprise/auditing.md

```diff
@@ -14,41 +14,42 @@ Auditing allows you to track important changes to your Grafana instance. By defa
 - A user failing to log in.
 
 ### Format
 
 Audit logs contain the following fields. The fields followed by **\*** are always available, the others depends on the type of action logged.
 
-| Field name | Type | Description |
-| ---------- | ---- | ----------- |
-| `timestamp`\* | string | The date and time the request was made, in coordinated universal time (UTC) using the [RFC3339](https://tools.ietf.org/html/rfc3339#section-5.6) format. |
-| `user`\* | object | Information about the user that made the request. At least one of the `UserID` / `ApiKeyID` fields will not be empty if `isAnonymous=false`. |
-| `user.userId` | number | ID of the Grafana user that made the request. |
-| `user.orgId`\* | number | Current organization of the user that made the request. |
-| `user.orgRole` | string | Current role of the user that made the request. |
-| `user.name` | string | Name of the Grafana user that made the request. |
-| `user.apiKeyId` | number | ID of the Grafana API key used to make the request. |
-| `user.isAnonymous`\* | boolean | `true` if an anonymous user made the request, `false` otherwise. |
-| `action`\* | string | The request action (eg. `create`, `update`, `manage-permissions`). |
-| `request`\* | object | Information about the HTTP request. |
-| `request.params` | object | Request path parameters. |
-| `request.query` | object | Request query parameters. |
-| `request.body` | string | Request body. |
-| `result`\* | object | Information about the HTTP response. |
-| `result.statusType`\* | string | `success` if the request action was successful, `failure` otherwise. |
-| `result.statusCode` | number | HTTP status of the request. |
-| `result.failureMessage` | string | HTTP error message. |
-| `result.body` | string | Response body. |
-| `resources` | array | Information about the resources that the request action impacted. Can be null for non-resource actions like `login` and `logout`. |
-| `resources[x].id`\* | number | ID of the resource. |
-| `resources[x].type`\* | string | Type of the resource (logged resources are: `alert`, `alert-notification`, `annotation`, `api-key`, `auth-token`, `dashboard`, `datasource`, `folder`, `org`, `panel`, `playlist`, `report`, `team`, `user`, `version`). |
-| `requestUri`\* | string | Request URI. |
-| `ipAddress`\* | string | IP address that the request was made from. |
-| `userAgent`\* | string | Agent through which the request was made. |
-| `grafanaVersion`\* | string | Grafana current version when this log is created. |
-| `additionalData` | object | Provide additional information on the request. For now, it's only used in `login` actions to log external user information if an external system was used to log in. |
+| Field name              | Type    | Description                                                                                                                                                                                                              |
```

Comment:
What changed in this file?

Reply:
This is an automatic linting update. I thought it was a new repo rule and would be applied for every table. If it's something on my side I can revert it. Can you confirm that tables should not have aligned `|`?

Reply:
Happy to follow protocol. I will have to look at these changes some other way than on a laptop screen because the lines are very long.

Reply:
It may be something on my side though, maybe @oddlittlebird can confirm if we should align the `|` in the tables or not?

And other than that, I didn't do any changes to the table content in this PR so it doesn't really need a review.

Reply:
I am all for standardization, and at the same time view 80+ character widths for lines intended for humans to read quite unfriendly. It introduces scrolling in the area of the content that is often reviewed (the long-form text).

Reply:
Note that my comments are non-blocking for this PR. If you need to merge, we can move this topic to a general linting topic.